smart-contracts/test/RelayPool/fees.hardhat.ts

```diff
@@ -67,11 +67,12 @@ describe('Fees', () => {
     const recipientAddress = ethers.Wallet.createRandom().address
 
     const amount = ethers.parseUnits('1')
-    const [, , , , , , bridgeFee] = await relayPool.authorizedOrigins(
-      10,
-      relayBridgeOptimism
+    expect(FRACTIONAL_BPS_DENOMINATOR).to.equal(
+      await relayPool.FRACTIONAL_BPS_DENOMINATOR()
     )
-    const fees = (amount * bridgeFee) / 10000n
+    expect(FRACTIONAL_BPS_DENOMINATOR / bridgeFee).to.equal(100) // make sure its 1%/100bps
+    const expectedFees = (amount * bridgeFee) / FRACTIONAL_BPS_DENOMINATOR
```

Comment:
Please put the _actual_ number here. No calculation!

```suggestion
    const expectedFees = (amount * bridgeFee) / FRACTIONAL_BPS_DENOMINATOR
```